docs/readmes/nms/dev_testing.md

```diff
@@ -0,0 +1,29 @@
+---
+id: dev_testing
+title: Testing Tips
+hide_title: true
+---
+
+# Testing Tips
+
+This guide covers tips for quickly validating NMS changes.
+
+## Run Test on the host
+
+There are some simple tests that you should run after making changes.
+
+### Eslint
+Run `cd ${MAGMA_ROOT}/nms/app/ && yarn run eslint` to test that there are no linter errors.
+You can also run `cd ${MAGMA_ROOT}/nms/app/ && yarn run eslint --fix` to fix automatically
+fixable linter errors.
+
+### Flow
+Run `cd ${MAGMA_ROOT}/nms/app/ && yarn run flow` to test that there are no type errors.
+
+### Unit Test
+Run `cd ${MAGMA_ROOT}/nms/app/ && yarn run test` to test that your changes have not broken existing
```

Comment:
All these "run x" commands are cd-ing to the same directory. Instead, just make a note near the top of this page that the working directory is "magma root /nms/app"